app/assets/javascripts/services/project_roles_svc.js

```diff
@@ -0,0 +1,28 @@
+samson.service('projectRolesService', function ($http) {
+
+    this.loadProjectRoles = function () {
+        return $http.get('/project_roles');
+    };
+
+    this.createProjectRole = function (project_role) {
+        var data = {
+            project_role: {
+                user_id: project_role.user_id,
+                project_id: project_role.project_id,
+                role_id: project_role.role_id
+            }
+        };
```

Comment:
That was my initial attempt, but I was having problems because, in the ruby controller, I only permit the following parameters: `user_id`, `project_id` and `role_id`. That's why I needed to create a different payload for this operation.

But you're right, I also didn't like this. The problem is when I initialise the JS controller: I always create this JSON object with the `id` attribute, which will contain the `id` of the association or `undefined` if it hasn't been created yet. 

I'm going to create an Angular factory to tackle this by building a proper Javascript object representing this data (ProjectRole) which, in it's turn, will know how to build the payload for a create and an update operation. I think I should have done this from the beginning. A reference for what I mean with this: https://medium.com/opinionated-angularjs/angular-model-objects-with-javascript-classes-2e6a067c73bc

Are you guys ok with that?